src/client/clientSimulation.ts

```diff
@@ -21,13 +22,37 @@ export default class ClientSimulation extends Simulation {
     super(map, updateStep, numPlayers, seed, { state: true });
   }
 
-  public update(body: Body, input: Input): void {
+  protected updateWave(_killed: number): void {
+    return;
+  }
+
+  private predictOtherPlayers(me: Id): void {
```

Comment:
`me` är inte jättebeskrivande?
```suggestion
  private predictOtherPlayers(ignore: Id): void {
```

Reply:
det är ju `me` och alla andra tänkte jag.